docs/quickstarts/1_client_credentials.rst

```diff
@@ -93,6 +93,13 @@ Loading the resource and client definitions happens in ``Startup.cs`` - the temp
             .AddInMemoryIdentityResources(Config.GetIdentityResources())
             .AddInMemoryApiResources(Config.GetApis())
             .AddInMemoryClients(Config.GetClients());
+        
+        // To know how to obtain the Environment property,
+        // take a look at the full Startup.cs file in our reporisory
```

Comment:
Spelling mistake in comment `reporisory` > `repository`

Reply:
Thanks. Fixed => [74ae3ed](https://github.com/IdentityServer/IdentityServer4/pull/3273/commits/74ae3eda0b5943cfcabe35cd3678aad02b8165b2)